### PR TITLE
terraform/gcp: Able to configure image type

### PR DESCRIPTION
**Pull request by cofyc**
### What problem does this PR solve? 

Sometimes users may favor non-COS node images. This adds variables to configure image type for TiDB node pools.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

### Does this PR introduce a user-facing change?:
 
 ```release-note
terraform/gcp: Able to configure node image types for PD/TiDB/TiKV node pools.
 ```

### CODE REVIEW

File: deploy/gcp/variables.tf [hunk 1]
```diff
@@ -80,6 +80,21 @@ variable "tikv_instance_type" {}
...
+
+variable "tidb_image_type" {
+  description = "TiDB image type, avaiable: UBUNTU/COS"
+  default = "COS"
+}
+
+variable "tikv_image_type" {
+  description = "TiKV image type, avaiable: UBUNTU/COS"
+  default = "COS"
+}
```
Comment (aylei) [inline deploy/gcp/variables.tf]: These variables should be added to `deploy/modules/gcp/tidb-cluster/variables.tf`

Comment (cofyc) [reply deploy/gcp/variables.tf]: OK, I'll fix it.

Comment (cofyc) [reply deploy/gcp/variables.tf]: fixed

**Pull request merged**